tools/test-cli-fatjar/test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -e
+set +x
+
+trap "cd $(pwd -P)" EXIT
+cd "$(dirname $0)"
+
+echo "Running CLI..."
+mvn exec:java -e -Dexec.mainClass=com.microsoft.playwright.CLI -Dexec.args="install"
```

Comment:
The last actions run failed with this error
```kotlin
╔══════════════════════════════════════════════════════════════════════════════════════════╗
║ Looks like Playwright was just installed or updated.                                     ║
║ Please run the following command to download new browsers:                               ║
║                                                                                          ║
║     mvn exec:java -e -Dexec.mainClass=com.microsoft.playwright.CLI -Dexec.args="install" ║
║                                                                                          ║
║ <3 Playwright Team                                                                       ║
╚══════════════════════════════════════════════════════════════════════════════════════════╝
```

Reply:
Oh, I know what's going on, in your TestApp call Driver.createAndInstall instead of Driver.ensureDriverInstalled so that it doesn't change Driver.instance and when you call Playwright.create it will actually try to do all initialization.

Reply:
I see what happened, I used false as the downloadBrowsers prarmeter in Driver.ensureDriverInstalled, should be fixed with the latest commit.

Reply:
I use `Driver.ensureDriverInstalled` instead of `Driver.createAndInstall` so that playwright will use the same driver.